test.sh

```diff
@@ -2,34 +2,86 @@
 set -e
 
 LOCAL_REPORTS_DIR="./reports"
-BUCKET_NAME="fcp-mpdp-performance-test-suite"
 
-echo "Starting fcp-mpdp-performance-test-suite Docker container..."
-docker compose up --build -d 
+# Clean up any existing reports using Docker (handles root-owned files)
```

Comment:
Needed to change this file quite a bit to support proper setup and cleanup of tests.  There was also a race condition between the container running the tests and the delete and cleanup of the folder.

Have moved the setup of S3 to localstack init script to simplify